README.md

```diff
@@ -51,7 +51,7 @@ to make these changes available in both projects.
 
 - Rate our Integreat app in the [PlayStore](https://play.google.com/store/apps/details?id=tuerantuer.app.integreat)
   and the [Apple App Store](https://apps.apple.com/ae/app/integreat/id1072353915).
-- Install [nodejs](https://nodejs.org/). At least v14.17 is required.
+- Install [nodejs](https://nodejs.org/). (Version >= 15.0 and <= 16.13.0 required)
```

Comment:
Why is it restricted not to be higher than `16.13.0`?

Reply:
Otherwise this error will occur: `TypeError: Cannot read properties of undefined (reading 'transformFile')`

Reply:
Did you find an issue for this?

Reply:
I think setting `export NODE_OPTIONS=--openssl-legacy-provider` fixes this issue https://github.com/nodejs/node/issues/40455